packages/web/src/entity/PublicDataset/index.ts

```diff
@@ -49,10 +49,12 @@ export const DatasetAnnotations = {
     DATASET_ID: new DatasetAnnotation("Dataset ID", "dataset_id"),
     DATASET_NAME: new DatasetAnnotation("Dataset name", "dataset_name", 50),
     DATASET_PATH: new DatasetAnnotation("File Path", "dataset_path"),
+    INDEX: new DatasetAnnotation("Index", "index", 1),
     DATASET_SIZE: new DatasetAnnotation("Size", "dataset_size", 78),
     DATASET_DESCRIPTION: new DatasetAnnotation("Short description", "description", 200),
     DOI: new DatasetAnnotation("DOI", "doi"),
     FILE_COUNT: new DatasetAnnotation("File count", "file_count", 89),
+    ORGANIZATION: new DatasetAnnotation("Organization", "organization", 114),
```

Comment:
unsure if these new annotations should also be added to the `PublicDatasetProps` interface above